keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -787,6 +787,10 @@ in terms of retriability and evolving Pod condition types
 to do not add any pod condition in this case. It should be re-considered in the
 future if there is a good motivating use-case.
 
+The reported issue which could be addressed by the new condition for exceeding
```

Comment:
I agree that this feature can go GA without needing to resolve the active deadline race with process exit in the kubelet.